src/__tests__/hooks/useHandleChangeSearch.test.ts

```diff
@@ -35,9 +33,7 @@ describe('Tests useHandleSearchHook', () => {
     const { result } = renderHook(() => useHandleChangeSearch(), {
       wrapper: StoreProvider,
     });
-    await act(async () => {
-      result.current.handleChangeSearch(searchState);
-    });
+    result.current.handleChangeSearch(searchState);
```

Comment:
`act` was useless here (and below): the hooks are not rendered so this never conflicts with react's render cycle.